app/src/main/java/io/homeassistant/companion/android/settings/qs/views/ManageTilesView.kt

```diff
@@ -0,0 +1,161 @@
+package io.homeassistant.companion.android.settings.qs.views
+
+import android.widget.Toast
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.Divider
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringArrayResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.graphics.drawable.DrawableCompat
+import androidx.core.graphics.drawable.toBitmap
+import androidx.fragment.app.FragmentManager
+import com.maltaisn.icondialog.IconDialog
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.database.AppDatabase
+import io.homeassistant.companion.android.database.qs.TileEntity
+import io.homeassistant.companion.android.settings.qs.ManageTilesViewModel
+
+@Composable
+fun ManageTilesView(
+    viewModel: ManageTilesViewModel,
+    iconDialog: IconDialog,
+    childFragment: FragmentManager
+) {
+    val context = LocalContext.current
+    var expandedTile by remember { mutableStateOf(false) }
+    var expandedEntity by remember { mutableStateOf(false) }
+    Column(modifier = Modifier.padding(20.dp)) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Text(
+                text = stringResource(R.string.tile_select),
+                fontSize = 15.sp,
+                modifier = Modifier.padding(end = 10.dp)
+            )
+            OutlinedButton(onClick = { expandedTile = true }) {
+                Text(
+                    viewModel.selectedTileName.value
+                )
+            }
+        }
+
+        DropdownMenu(expanded = expandedTile, onDismissRequest = { expandedTile = false }) {

```

Comment:
Suggestion: wrap the `OutlinedButton` for selecting the tile with a `Box`, and move the dropdown menu inside there, because now the dropdown doesn't align with the button. See [Wear OS template tile settings as an example](https://github.com/home-assistant/android/blob/b7f6a7322cfc156f2854513bb5fa32d7fdbe89af/app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTile.kt#L92-L113).
![image](https://user-images.githubusercontent.com/8148535/150851534-e165e0b2-0886-44b3-a194-fc8a9a4e709f.png)